code/pcxutils/pcxutils.cpp

```diff
@@ -310,7 +310,22 @@ int pcx_read_bitmap( const char * real_filename, ubyte *org_data, ubyte *pal, in
 			if ( col < xsize ) {
 				// 8-bit PCX reads
 				if ( byte_size == 1 ) {
-					*pixdata++ = data;
+					auto pixel_val = data;
+					if (!mask_bitmap) {
+						// 8 bit-per-pixel aa bitmaps are a bit special since they only use values in the range [0, 15] where 15 wraps
+						// around back to 0. Since the rest of the code expects the value to be in the range [0, 255] the pixel value
+						// needs to be adjusted here. By multiplying the value with 17 the original range [0, 15] is mapped to [0, 255]
+						// This only applies to bitmaps that are not used as masks since mask bitmaps use higher values
+						// to indicate their mask area
+						if (data > 15) {
+							pixel_val = 0;
+						} else if (data == 15) {
+							pixel_val = 17;
```

Comment:
So... 16 or greater wraps around to 0, but 15 is treated like 1?

Reply:
Exactly! Welcome to the world of legacy color conversions where the knowledge of why it was done that way has been lost for a long time...

I derived this function from the old translation map which is build by this code: https://github.com/scp-fs2open/fs2open.github.com/blob/fb4169ba9fce882e1a436a4fce67947270aa85e1/code/graphics/opengl/gropengltexture.cpp#L174-L178
Line 178 is the reason for the special case above since the entry for the pixel value 15 is set to the value of 1 which is always `17`. I have no idea why it is done that way, I only replicated it in this code...

Reply:
Well, okay then... guess it's backwards-compatible.